yangvalidator/v2/views.py

```diff
@@ -193,7 +193,10 @@ def validate_doc(request: WSGIRequest):
     url = ''
     if doc_type == 'draft':
         draft_dir = os.path.join(ietf_dir, 'my-id-archive-mirror')
-        matching_drafts = fnmatch.filter(os.listdir(draft_dir), f'{doc_name}*.txt')
+        try:
+            matching_drafts = fnmatch.filter(os.listdir(draft_dir), f'{doc_name}*.txt')
+        except FileNotFoundError:
+            matching_drafts = []
```

Comment:
maybe it would be better to check if the directory exists first and then use ```fnmatch.filter```? I mean:
```
matching_drafts = fnmatch.filter(os.listdir(draft_dir), f'{doc_name}*.txt') if os.path.exists(draft_dir) else []
```
so we can avoid this ```try-except clause

Reply:
sure, updated

Reply:
Problem with tests, reverting back to try-except implementation